README.md

```diff
@@ -126,6 +128,10 @@ Browse the documentation to learn more. Here are some topics you may be interest
 
 - Nydus can also run with macfuse(a.k.a osxfuse).For more details please read [nydus with macos](./docs/nydus_with_macos.md).
 
+## Run eStargz image
+
+Use the containerd remote snapshotter plugin [nydus-snapshotter](https://github.com/containerd/nydus-snapshotter), which supports both nydus image runs and direct support for [eStargz](https://github.com/containerd/stargz-snapshotter) image runs with appending `--enable-stargz` command line option.
```

Comment:
which supports running both Nydus images and [eStargz](https://github.com/containerd/stargz-snapshotter) images with `--enable-stargz` command line option appended.